### PR TITLE
changes to improve event system based on new web ui devs requests

### PR DESCRIPTION
**Pull request by douglascamata**
- added encounter_id, latitude, longitude and pokemon_id to pokemon caught, vanished and appeared events.

- data structure of events broadcasted by the bots now is the following:

```
{
    "event": event_name,
    "account": bot_account,
    "data": event_parameters
}
```

- position update event being sent at debug level for each step in the `StepWalker`

- `get_player_info` remote command now includes player and inventory info

@brantje: use this PR to keep working on the web ui and we discuss further modifications here.

### CODE REVIEW

File: brantje.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# coding: utf-8
```
Comment (TheSavior) [inline brantje.py]: Did this mean to be committed?

**Comment by Jasperrr91** [comment]
I can haz merge?

**Comment by brantje** [comment]
:+1:

File: brantje.py (hunk 1 above)
Comment (douglascamata) [inline brantje.py]: I'll remove in future commit, sorry :P

**Pull request merged**